optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -1286,6 +1308,10 @@ def _reorder_cache(past, beam_idx) -> Tuple[Tuple[torch.FloatTensor]]:
             )
         return reordered_past
 
+    def can_generate(self):
```

Comment:
Could this be put in the `ORTModelForConditionalGeneration`?

Reply:
done